src/test/kotlin/se/uulm/snowballr/backend/repository/ReviewTableRepoTest.kt

```diff
@@ -86,7 +86,7 @@ class ReviewTableRepoTest : RepositoryTest(arrayOf(ReviewTable, ProjectTable, Pr
         }
 
         @Test
-        fun `When the project paper has no reviews and other reviews exist, then only the project paper reviews are returned`() =
+        fun `When the project paper has reviews but other reviews exist too, then only the project paper reviews are returned`() =
```

Comment:
Since there's no contradiction in that, I would choose "and".
```suggestion
        fun `When the project paper has reviews and other reviews exist too, then only the project paper reviews are returned`() =
```

Reply:
I changed it and squashed it into the commit where I fixed the other test description "mistakes".